### DISCUSSION TITLE
Where is "Wind" Theme for PrimeVue?

### DISCUSSION THREAD

**Discussion by FabianMontoya** [primefaces/community#1512] [PrimeVue] [answered]
I'm looking into the[ primevue's theme list](https://primevue.org/theming/#builtinthemes) and i can't find the "**Wind**" theme used in [tailwind.primevue.org](https://tailwind.primevue.org), where can i use this theme?, I'm in a vite project.
![Screenshot 2024-03-25 at 10 42 51](https://github.com/primefaces/community/assets/37662924/b3401a91-0361-4fcb-92d4-38b847ff0f19)

**Comment by tugcekucukoglu** [comment, marked as answer]
Wind is not a PrimeVue theme but is a built-in preset used with Tailwind. And can be downloaded here https://tailwind.primevue.org/builder/
